Tests/SocketTests/SocketTests.swift

```diff
@@ -1125,6 +1125,18 @@ class SocketTests: XCTestCase {
 		}
 	}
 	
+	func testFDSetBitFields() {
+		var fdSet = fd_set()
+		fdSet.zero()
+		
+		for i: Int32 in 0...128 {
```

Comment:
The default number of max open file descriptors on Linux is **1024**. On macOS, the default is **unlimited**.  I think checking for 1024 should be enough but 128 is a bit too little... 😉  Making this change after the merge.